src/coreclr/scripts/superpmi-setup.py

```diff
@@ -0,0 +1,395 @@
+#!/usr/bin/env python3
+#
+## Licensed to the .NET Foundation under one or more agreements.
+## The .NET Foundation licenses this file to you under the MIT license.
+#
+##
+# Title               : superpmi-setup.py
+#
+# Notes:
+#
+# Script to setup directory structure to perform SuperPMI collection in CI.
```

Comment:
It would be useful for this top comment to be more descriptive: what the inputs, outputs, what does it need to do to "setup directory structure..."?